src/Tests/Guzzle/Middleware/GuzzleRequestIdInjectorTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupBundle\Guzzle\Subscriber;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use Mockery as m;
+use PHPUnit_Framework_TestCase as UnitTest;
+use Surfnet\StepupBundle\EventListener\RequestIdRequestResponseListener;
+use Surfnet\StepupBundle\Guzzle\Middleware\GuzzleRequestIdInjector;
+use Surfnet\StepupBundle\Request\RequestId;
+use Surfnet\StepupBundle\Request\RequestIdGenerator;
+
+class GuzzleRequestIdInjectorTest extends UnitTest
+{
+    /**
+     * @group Guzzle
+     */
+    public function testItSetsTheRequestIdAsHeader()
```

Comment:
Small nitpick: this is inconsistent with the capitalization of methods in other test classes.